windows-agent/internal/proservices/landscape/executor_test.go

```diff
@@ -149,7 +150,13 @@ func TestInstall(t *testing.T) {
 	ctx, cancel := context.WithCancel(context.Background())
 	t.Cleanup(cancel)
 
-	fileServerAddr := mockRootfsFileServer(t, ctx)
+	fileServerAddr, closer := mockRootfsFileServer(t, ctx)
+	t.Cleanup(func() {
+		err := closer.Close()
+		if err != nil {
+			t.Logf("Could not close mock fileserver: %v", err)
```

Comment:
Please prefix with "Teardown:" so that we can spot this is not the test call itself.